tests/unit/20_storage.bats

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+setup() {
+  create_app
+}
+
+teardown() {
+  destroy_app
+}
+
+@test "(storage) storage:list" {
+  run /bin/bash -c "dokku storage:mount $TEST_APP /tmp/mount:/mount"
+  run /bin/bash -c "dokku storage:list $TEST_APP | grep -qe '^\s*/tmp/mount:/mount'"
+  echo "output: "$output
+  echo "status: "$status
+  assert_success
+}
+
+@test "(storage) storage:mount" {
```

Comment:
Yeah I was literally thinking you could delete lines 13-20 and change line 21 to say `@test "(storage) storage:mount, storage:list"`